CONTRIBUTING.md

```diff
@@ -45,6 +45,12 @@ Specifically:
 
 Refer to [Google's HTML/CSS Style Guide][google_style_guide] for all other formatting rules.
 
+# Contributing Code
+
+1. Create a new git branch specific to your change instead of to making your commits in the master branch.
+1. Create pull reuqest from that branch.
```

Comment:
Tiny typo on this line. Should be "request"

Reply:
Thank you @AWegnerGitHub